inginious/frontend/templates/course_admin/edit_tabs/tags.html

```diff
@@ -0,0 +1,148 @@
+$def with (course,task_data)
+
+$#
+$# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
+$# more information about the licensing of this file.
+$#
+
+<ul class="nav nav-tabs">
+    <li class="active"><a data-toggle="tab" href="#home">$:_("Tag editor")</a></li>
+    <li><a data-toggle="tab" href="#menu1">$:_("Import tags")</a></li>
+</ul>
+
+<!-- Menu Tag Editor -->
+<div class="tab-content">
+<div id="home" class="tab-pane fade in active">
+
+<div class="form-group">
+    <table class="table-striped" id="table">
+    <thead>
+        <tr>
+            <th class="col-sm-2">$:_("Id")</th>
+            <th class="col-sm-3">$:_("Name")</th>
+            <th class="col-sm-4">$:_("Description")</th>
+            <th class="col-sm-1">$:_("Show to students")</th>
+            <th class="col-sm-2">$:_("Type ")</th>
+        </tr>
+    </thead>
+    <tbody>
+    $ i=-1
+    $for tag in task_data.get('tags', {}):
+        $ i=i+1
+        $ id = task_data.get('tags')[tag]["id"] if "id" in task_data.get('tags')[tag] else "unknow_id"
+        $ name = task_data.get('tags')[tag]["name"] if "name" in task_data.get('tags')[tag] else "Unknow name"
+        $ description = task_data.get('tags')[tag]["description"] if "description" in task_data.get('tags')[tag] else ""
+        $ type = task_data.get('tags')[tag]["type"] if "type" in task_data.get('tags')[tag] else 0
+        $ visible = task_data.get('tags')[tag]["visible"] if "visible" in task_data.get('tags')[tag] else False
```

Comment:
You should replace the loop by something like ``for key, tag in task_data.get('tags', {}).items():``